streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestDriver.java

```diff
@@ -470,6 +461,18 @@ public static VerificationResult verify(final String kafka,
         return verificationResult;
     }
 
+    private static Map<String, Set<Number>> parseRecordsForEchoTopic(final Map<String, Map<String, LinkedList<ConsumerRecord<String, Number>>>> events) {
+        return events.containsKey("echo") ?
```

Comment:
nit: This is along expression, I'd prefer to write this with an if-construct, but I won't insist.

Reply:
Thank you for your review!

This is fixed in latest commit.